### PR TITLE
U2 wrong orientation

### PR DESCRIPTION
**Pull request by CzarofAK**
Changed orientation of the U2 to 90°

### CODE REVIEW

**Comment by CzarofAK** [comment]
or does it need to be 270°?

**Comment by stuartpittaway** [comment]
Hello, this is quite an old design and been in use for a long time with the files as they were, I'm surprised there is an issue.

Are you aware this version is obsolete? The New controller version is the latest.

**Comment by CzarofAK** [comment]
Hello Stuart, yes I am aware. But i really like this slim version and fits perfectly in my design. As long as it is working with my new v4.4 boards, its all good.
I guess the orientation should be 270° instead of 90°.

**Comment by CzarofAK** [comment]
Unbenannt
That's how it looks on the JLC homepage, if I use the Github file.
Beside that, it passed the check and is now already in production.

**Comment by stuartpittaway** [comment]
Ok, definitely needs rotating, surprised nobody else mentioned this

**Comment by CzarofAK** [comment]
i am eager to see how mine will arrive, as i ordered with the wrong orientation. And as i wrote above, it passed the JLC internal quality check...

**Comment by CzarofAK** [comment]
JLC correcte the orientation of U2 themselves

File: ESP8266ControllerCircuit/ESP8266ControllerCircuit_cpl_jlc.csv [hunk 1]
```diff
@@ -5,5 +5,5 @@ R2,4K7,R_0805_HandSoldering,19.500000,19.062500,180.000000,top
 R3,220R,R_0805_HandSoldering,19.500000,15.625000,180.000000,top
 R4,4K7,R_0805_HandSoldering,19.500000,12.187500,180.000000,top
 R5,4K7,R_0805_HandSoldering,19.500000,8.750000,180.000000,top
-U2,EL3H7(B)(TA)-G,SOP-4_4.4x2.8mm_Pitch1.27mm,57.250000,4.500000,180.000000,top
```
Comment (stuartpittaway) [inline ESP8266ControllerCircuit/ESP8266ControllerCircuit_cpl_jlc.csv:8]: Just tested it, and it needs to be 270 degrees
U2,EL3H7(B)(TA)-G,SOP-4_4.4x2.8mm_Pitch1.27mm,57.250000,4.500000,270.000000,top

**Comment by stuartpittaway** [comment]
Thank you for the PR/change

**Pull request merged**